packages/graph-explorer/src/connector/gremlin/types.ts

```diff
@@ -39,7 +39,7 @@ export type GVertexProperty = {
 export type GProperty = {
   "@type": "g:Property";
   "@value": {
-    id: GInt32;
+    id?: GInt32;
```

Comment:
From my inspection of gremlin responses, this `id` field is not always provided. So making it optional.